modules/src/lib.rs

```diff
@@ -26,6 +27,7 @@
 //! - ICS 26: Routing
 //! - Applications:
 //!    - ICS 20: Fungible Token Transfer
+extern crate no_std_compat as std;
```

Comment:
Can we use `core` and `alloc` directly without going through `no_std_compat`? The library has not been updated for a while, and very unfortunately the author seems to have [passed away](https://www.redox-os.org/news/open-source-mental-health/).

Reply:
I think it is necessary to have a library that can switch between std and no_std. If you think switching to `no-std-compat` doesn't work, what do you think about switching to `sp-std`?